package.json

```diff
@@ -32,6 +32,17 @@
   "bugs": {
     "url": "https://github.com/darkreader/darkreader/issues"
   },
+  "keywords": [
+    "dark-theme",
+    "chrome-extension",
```

Comment:
`chrome-extension` and `firefox-addon` can be removed as this is a NPM package and not the full extension.

Reply:
Done in e71245a